stattutor/public/js/ctat.min.js

```diff
@@ -0,0 +1,1126 @@
+/*
+ MIT License - https://bitbucket.org/pellepim/jstimezonedetect/src/default/LICENCE.txt
+
+ For usage and examples, visit:
+ http://pellepim.bitbucket.org/jstz/
+
+ Copyright (c) Jon Nylander
+*/
+var $jscomp={scope:{}};$jscomp.defineProperty="function"==typeof Object.defineProperties?Object.defineProperty:function(a,b,e){if(e.get||e.set)throw new TypeError("ES3 does not support getters and setters.");a!=Array.prototype&&a!=Object.prototype&&(a[b]=e.value)};$jscomp.getGlobal=function(a){return"undefined"!=typeof window&&window===a?a:"undefined"!=typeof global&&null!=global?global:a};$jscomp.global=$jscomp.getGlobal(this);$jscomp.SYMBOL_PREFIX="jscomp_symbol_";
```

Comment:
Can you include a non-minified copy of this file in the repo?
It's impossible to review/edit as-is.